### PR TITLE
Add support for logpoints

### PR DESCRIPTION
**Pull request by puremourning**
Log points are added when `logMessage` is in the extra breakpoints options.

These work in codelldb, but in vscode-cpptools it seems that you have to only add them _after_ starting debugging (don't seem to work if added beforehand). So we use codelldb in the tests, which is a good thing to be doing anyway.

### CODE REVIEW

**Comment by puremourning** [comment]
This change is [Reviewable](https://reviewable.io/reviews/puremourning/vimspector/462)

File: README.md [hunk 1]
```diff
@@ -880,8 +880,12 @@ dictionary of options. The dictionary can have the following keys:
   times the breakpoint should be ignored. Should (probably?) not be used in
   combination with `condition`. Not supported by all debug adapters. For
   example, to break on the 3rd time hitting this line, enter `3`.
+* `logMessage`: An optional string to make this breakpiont a "logpoint" instead.
```
Comment (mark2185) [inline README.md:883]: `breakpiont -> breakpoint`

File: README.md [hunk 2]
```diff
@@ -880,8 +880,12 @@ dictionary of options. The dictionary can have the following keys:
...
+  That is, when triggered, a this message is printed to the console rather than
```
Comment (mark2185) [inline README.md:884]: "when triggered, ~~a~~ this message"

File: README.md (hunk 1 above)
Comment (puremourning) [reply README.md:883]: thanks

File: README.md (hunk 2 above)
Comment (puremourning) [reply README.md:884]: thanks

**Review by puremourning — COMMENTED** [review]
:lgtm:

> Reviewed 2 of 12 files at r1, 10 of 10 files at r3, all commit messages.
> Dismissed @mark2185 from 2 discussions.
> *[Reviewable](https://reviewable.io/reviews/puremourning/vimspector/462#-:-Mk6rxhHAKcOEhC2rH7E:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @puremourning)

**Pull request merged**